casper/src/main/scala/coop/rchain/casper/PrettyPrinter.scala

```diff
@@ -0,0 +1,51 @@
+package coop.rchain.casper
+
+import com.google.protobuf.ByteString
+import coop.rchain.casper.protocol.Resource.ResourceClass.{
+  ConsumeResource,
+  ProduceResource,
+  StoppedResource
+}
+import coop.rchain.casper.protocol.{BlockMessage, Resource, Stopped}
+import scalapb.GeneratedMessage
+import coop.rchain.crypto.codec._
+
+class PrettyPrinter {
```

Comment:
Why is this a `class` if it only contains functions? Maybe `object` will be enough?

Reply:
Done